packages/replay_cubit/lib/src/replay_cubit.dart

```diff
@@ -41,13 +41,19 @@ part 'change_stack.dart';
 /// * [Cubit] for information about the [ReplayCubit] superclass.
 ///
 /// {@endtemplate}
-abstract class ReplayCubit<State> extends Cubit<State> {
+abstract class ReplayCubit<State> extends Cubit<State> with ReplayMixin<State> {
   /// {@macro replay_cubit}
-  ReplayCubit(State state, {int limit})
-      : _changeStack = _ChangeStack<State>(limit: limit),
-        super(state);
+  ReplayCubit(State state, {int limit}) : super(state) {
+    this.limit = limit;
+  }
+}
+
+mixin ReplayMixin<State> on CubitStream<State> {
```

Comment:
Is this supposed to be ReplayCubitMixin instead of ReplayMixin?

Reply:
Need to update the naming in the docs but I was thinking of ReplayMixin because I wanted it to be compatible with bloc as well. Thoughts?

Reply:
I like it being compatible with both, so ReplayMixin makes sense

Reply:
That seems good to me! Just wanted to make sure it wasn't an accident 😉

Reply:
Naturally we would have HydratedMixin as well 💪🏻